demo/extra.cc

```diff
@@ -232,24 +236,67 @@ void test_wallet_connect() {
           client->sign_personal_blocking("hello", result.addresses[0].address);
       cout << "signature=" << bytes_to_hex_string(sig1).c_str() << std::endl;
       cout << "signature length=" << sig1.size() << endl;
-    } else {
-      /* legacy eth sign */
-      WalletConnectTxLegacy info;
-      info.to =
-          String(std::string("0x") +
-                 address_to_hex_string(result.addresses[0].address).c_str());
+    }
+
+    if (test_basic) {
+      std::string fromaddress = getenv("MYFROMADDRESS");
+      cout << "mycronosrpc=" << mycronosrpc << endl;
+      cout << "fromaddress=" << fromaddress << endl;
+      std::string toaddress = getenv("MYTOADDRESS");
+      cout << "toaddress=" << toaddress << endl;
+      std::string mynonce =
+          org::defi_wallet_core::get_eth_nonce(fromaddress.c_str(), mycronosrpc)
+              .c_str();
+      cout << "nonce=" << mynonce << endl;
+      WalletConnectTxEip155 info;
+      info.to = toaddress;
       info.gas = "21000";             // gas limit
       info.gas_price = "10000";       // gas price
-      info.value = "100000000000000"; // 0.0001 eth
+      info.value = "100000000000321"; // 0.0001 eth
```

Comment:
the comment doesn't match the new value or why was it updated?

Reply:
ok